### DISCUSSION TITLE
Using mermaid for diagrams in GitHub

### DISCUSSION THREAD

**Discussion by wavingtowaves** [SORTEE-Github-Hackathon/manuscript#86] [Ideas]
Hi everyone,

Not sure if this will be useful for any of the figures we are planning to include in the manuscript, but either way there's a nice way of creating flowcharts/diagrams in Markdown using [mermaid](https://github.com/mermaid-js/mermaid#readme). 

Saw this [blog post from GitHub](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/). The associated documentation for the javascript/markdown tool that could help us generate some ideas for formatting figures as well.

```mermaid
  graph TD;
      A--> B;
      A-->C;
      B-->D;
      C-->D;
```

**Comment by Aariq** [comment]
OOh, nice.  Could be useful!
